### PR TITLE
css selectors

### PR DESCRIPTION
**Pull request by KarenCelis**

### CODE REVIEW

File: buy-shirt.spec.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    cy.get('#email').type('aperdomobo@gmail.com')
+    cy.get('#passwd').type('WorkshopProtractor')
+
+    // Debes completar la prueba ...
+
+    cy.get('#center_column > div > p > strong')
+      .should('have.text', 'Your order on My Store is complete.')
+  });
+});
+
```
Comment (holgiosalos) [inline buy-shirt.spec.ts]: delete spaces in line 19

**Comment by renardete** [comment]
If the PR is not ready, you could convert it to draft PR.

File: buy-shirt.spec.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    cy.visit('http://automationpractice.com/')
+    cy.get('#block_top_menu > ul > li:nth-child(3) > a').click()
+    cy.get('#center_column a.button.ajax_add_to_cart_button.btn.btn-default').click()
+    cy.get('[style*="display: block;"] .button-container > a').click()
+    cy.get('.cart_navigation span').click()
+
+    cy.get('#email').type('aperdomobo@gmail.com')
+    cy.get('#passwd').type('WorkshopProtractor')
+
+    // Debes completar la prueba ...
```
Comment (holgiosalos) [inline buy-shirt.spec.ts]: In this step the idea is to complete the missing steps to have a successful execution, check message on slack:
https://psl-techtalks.slack.com/archives/C01QA81V29F/p1616800059027300

image

**Review by holgiosalos — CHANGES_REQUESTED** [review]
Please locate the `buy-shirt.spec.ts` file in the `cypress/integration` folder.

**Review by holgiosalos — APPROVED** [review]
Awesome job!

**Pull request merged**